### PR TITLE
add a unit test for AvailableClusterInvoker

### PR DESCRIPTION
**Pull request by hadley0828**
## What is the purpose of the change
issue #7882

## Brief changelog
add AvailableClusterInvokerTest.java

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GitHub issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [x] Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Check if is necessary to patch to Dubbo 3 if you are work on Dubbo 2.7
- [x] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [x] Add some description to [dubbo-website](https://github.com/apache/dubbo-website) project if you are requesting to add a feature.
- [x] GitHub Actions works fine on your own branch.
- [x] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: dubbo-cluster/src/test/java/org/apache/dubbo/rpc/cluster/support/AvailableClusterInvokerTest.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        given(dic.getInterface()).willReturn(AvailableClusterInvokerTest.class);
+
+        invocation.setMethodName("method1");
+
+        invokers.add(invoker1);
+        invokers.add(invoker2);
+        invokers.add(invoker3);
+    }
+
+    private void resetInvokerToException() {
```
Comment (icankeep) [inline dubbo-cluster/src/test/java/org/apache/dubbo/rpc/cluster/support/AvailableClusterInvokerTest.java:63]: unused method should remove

**Review by icankeep — COMMENTED** [review]
it would be better to add a case that invoker1 is unavailable

File: dubbo-cluster/src/test/java/org/apache/dubbo/rpc/cluster/support/AvailableClusterInvokerTest.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.dubbo.rpc.cluster.support;
+
+import org.apache.dubbo.common.URL;
+import org.apache.dubbo.rpc.*;
```
Comment (chickenlj) [inline dubbo-cluster/src/test/java/org/apache/dubbo/rpc/cluster/support/AvailableClusterInvokerTest.java:20]: avoid using import *

**Review by chickenlj — CHANGES_REQUESTED** [review]

**Comment by chickenlj** [comment]
Closed no feedbacks for days.

**Pull request closed without merging**